Gemfile

```diff
@@ -1,72 +1,80 @@
-source "https://rubygems.org"
-git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+source 'https://rubygems.org'
+git_source(:github) { |repo| `https://github.com/#{repo}.git` }
```

Comment:
Was this change recommended bay Rubocop 🤔 ? We commonly use double quotes for string interpolation

Reply:
![Captura de Pantalla 2022-06-07 a la(s) 2 33 00 p m](https://user-images.githubusercontent.com/106546244/172466915-5b9f88ce-40e7-4eb8-8be3-2f7b2417ac22.png)

Yes, I saw it strange but that's what rubocop said